dev-libs/rocksdb/rocksdb-9.6.1.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="A persistent key-value store for fast storage"
+DESCRIPTION="RocksDB is an embeddable persistent key-value store for fast storage.
+
+* Uses a log structured database engine, written entirely in C++, for \
+maximum performance. Keys and values are just arbitrarily-sized byte streams.
+* Is optimized for fast, low latency storage such as flash drives and \
+high-speed disk drives. RocksDB exploits the full potential of high read/write \
+rates offered by flash or RAM.
+* Is adaptable to different workloads.
+* Provides basic operations such as opening and closing a database, reading \
+and writing to more advanced operations such as merging and compaction filters."
+HOMEPAGE="https://rocksdb.org"
+COPYRIGHT="2011 Facebook, Inc."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/facebook/rocksdb/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="98cf497c1d6d0a927142d2002a0b6b4816a0998c74fda9ae7b1bdaf6b784e895"
+SOURCE_FILENAME="v$portVersion.tar.gz"
+SOURCE_DIR="rocksdb-$portVersion"
+PATCHES="rocksdb-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	rocksdb$secondaryArchSuffix = $portVersion
+	lib:librocksdb$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgflags$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	rocksdb${secondaryArchSuffix}_devel = $portVersion
+	devel:librocksdb${secondaryArchSuffix}
+	"
+REQUIRES_devel="
+	rocksdb$secondaryArchSuffix == $portVersion base
+	"
+
+PROVIDES_tools="
+    rocksdb${secondaryArchSuffix}_tools = $portVersion
+    cmd:cache_bench
+    cmd:db_bench
+    cmd:ldb
+    cmd:sst_dump
+    "
+REQUIRES_tools="
+    haiku$secondaryArchSuffix
+    rocksdb$secondaryArchSuffix == $portVersion base
+    lib:libgflags$secondaryArchSuffix
+    "
```

Comment:
Looks like there are spaces used here instead of one tab?

Reply:
Fixed.